tfgrid3/mastodon/Dockerfile

```diff
@@ -1,42 +1,59 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates curl gnupg lsb-release && \
+RUN apt-get update && apt-get install -y \
+    curl \
+    gnupg \
+    lsb-release \
+    wget \
+    apt-transport-https \
+    ca-certificates \
+    ufw \
+    vim \
```

Comment:
is the vim pkg needed?

Reply:
removed, i was installing this inside the container to debug.